### PR TITLE
Summary expansion concept - bar with Carousel 

### PR DESCRIPTION
**Pull request by rossbulat**
This PR demonstrates an enhanced summary section in the form of a unified panel, with an optional carousel built in allowing us to show stats that are currently hidden on smaller screens. 

I feel this is the direction the summaries should be going in whereby no stats are sacrificed on smaller devices.

Some UI has been slightly decreased in size for a more balanced font sizing.

Note:  Unfinished - concept only 

summarybar

If positive feedback is received I would then like to explore dynamic slides depending on window width.

Carousel used: 
[Pure React Carousel](https://www.npmjs.com/package/pure-react-carousel). Keeping components lightweight with great customisability.

### CODE REVIEW

File: packages/apps/webpack.config.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ function createWebpack ({ alias = {}, context, name = 'index' }) {
     plugins: plugins.concat([
       isProd
         ? null
-        : new WebpackPluginServe({
+        : new git({
```
Comment (jacogr) [inline packages/apps/webpack.config.js]: Not 100% sure about this part ;)

File: packages/ui-app/src/styles/app.css [hunk 2]
```diff
@@ -121,21 +121,75 @@ summary {
...
   justify-content: space-between;
   flex-wrap: wrap;
 }
 
-summary article {
-  color: rgba(0, 0, 0, 0.6);
-  flex: 0 1 auto;
-  text-align: left;
-}
+.summary--fullWidth {
```
Comment (jacogr) [inline packages/ui-app/src/styles/app.css]: Add the specific styles to `component.css` rather.

File: packages/ui-app/src/styles/app.css [hunk 3]
```diff
@@ -121,21 +121,75 @@ summary {
   text-align: center;
 }
 
-summary {
+summary,
+
+.isSummary {
```
Comment (jacogr) [inline packages/ui-app/src/styles/app.css]: This classname seems a bit _too_ generic for global use?

**Comment by rossbulat** [comment]
Yes, thank you for the feedback. 

The primary use case is for smaller screens where not all stats are visible, and are currently being hidden completely. 

The carousel would be disabled completely where all the stats fit the screen width. 

I am aware of the generic class names, this was just to demonstrate the concept. Apologies for not being clearer in the original post.

**Comment by jacogr** [comment]
No, you did say - I just wanted to be 100% when I looked at it. (And it does say "draft" as well :))

**Comment by rossbulat** [comment]
😄 To be fair I think the global screen size support OCD kicked in to work on this issue. But nonetheless will experiment with the dynamic slides.

**Pull request closed without merging**